src/mlpack/methods/ann/activation_functions/softplus_function.hpp

```diff
@@ -0,0 +1,114 @@
+/**
+ * @file softplus_function.hpp
+ * @author Vivek Pal
+ *
+ * Definition and implementation of the softplus function.
+ *
```

Comment:
Can you cite the "Incorporating Second-Order Functional Knowledge for Better Option Pricing" by C. Dugas et al. paper that introduced the softplus function? Take a look at the softsign function header for an example.